### ISSUE TITLE
Document env vars in obc-peer -h

### ISSUE THREAD

**Issue by msackman** [hyperledger-archives/fabric#867]
`obc-peer`'s help gives no way to discover the environment variables that it looks for. Grepping the codebase for `os.Getenv` doesn't reveal what I'm looking for, and even grepping for things like `OPENCHAIN_PEER_ID` doesn't find anything hopeful, so I _must_ be missing something as I can't figure out how these env vars get picked up by the binary, though they certainly do.

In general, it's good practise for binaries that accept parameters through the environment to also provide switches to provide them on the command line and to document them.

**Comment by msackman** [comment]
Yup, finally tracked this down through using the source. Your use of `viper`, with a string Replacer and `AutomaticEnv` made this pretty hard to figure out. Knowing this, I can now find all the config params with

```
obc-peer$ grep -ri viper.get . | grep '.go:' | sed -e 's/^.*viper\.Get[^"]\+"//' -e 's/".*//' | sort | uniq
```

which reveals there are 80 of them!

**Comment by bcbrock** [comment]
Maybe another way to solve this would be to state a requirement that all configuration parameters be included and documented in the *.yaml files with their default values. I'm not sure how to enforce this though, but your script above might be part of the solution.

**Comment by binhn** [comment]
Probably that would still be acceptable as I know some commands have tons of options (such as cf from bluemix). It's a matter of organizing the output.
